build.cake

```diff
@@ -104,7 +104,7 @@ Task("Externals-Android")
     var pushLib = "appcenter-push-release";
     var pushLibFile = $"{AndroidExternals}/{pushLib}.aar";
     var pushLibUnzippedPath = $"{AndroidExternals}/{pushLib}";
-    var manifestUpdateFile = "SDK/AppCenterPush/Microsoft.AppCenter.Push.Android.Bindings/Properties/AndroidManifest.xml";
+    var manifestUpdateFile = "SDK/AppCenterPush/Microsoft.AppCenter.Push.Android.Bindings/AndroidManifest.xml";
```

Comment:
This was causing new version of Visual Studio for Mac to believe the library project was an application and was updating the csproj...